content/docs/command-reference/commit.md

```diff
@@ -20,9 +20,10 @@ positional arguments:
 The `dvc commit` command is useful for several scenarios, when data already
 tracked by DVC changes: when a [stage](/doc/command-reference/run) or
 [pipeline](/doc/command-reference/dag) is in development/experimentation; when
-manually editing or generating DVC <abbr>outputs</abbr>; or to force update the
-`dvc.lock` or `.dvc` files without reproducing stages or pipelines. These
-scenarios are further detailed below.
+force updating the `dvc.lock` or `.dvc` files without reproducing stages or
+pipelines; when adding already existing <abbr>outputs</abbr> to a stage that
+missing from `dvc.yaml`; or when manually editing or generating DVC outputs.
+These scenarios are further detailed below.
```

Comment:
OK to reorder these but adding another sentence here is TMI. It's already a very long/hard-to-read paragraph. Let's merge the 2 last cases as s single sentence similar to the original version. BTW please don't remove `<abbr>` tags unless there's a reason. Those make tooltips on the website:

![image](https://user-images.githubusercontent.com/1477535/96513990-a4ee3b00-1228-11eb-8197-0bc0718125cb.png)
> https://dvc.org/doc/command-reference/commit

Reply:
I still see the new separate sentence there, BTW. Let's merge those last 2 sentences before "These scenarios..." somehow please.